moveit_ros/planning_interface/common_planning_interface_objects/include/moveit/common_planning_interface_objects/common_objects.h

```diff
@@ -56,7 +56,8 @@ robot_model::RobotModelConstPtr getSharedRobotModel(const std::string& robot_des
   @param tf
   @return
  */
-planning_scene_monitor::CurrentStateMonitorPtr getSharedStateMonitor(const robot_model::RobotModelConstPtr& kmodel,
+planning_scene_monitor::CurrentStateMonitorPtr getSharedStateMonitor(const std::string& robot_description,
+                                                                     const robot_model::RobotModelConstPtr& kmodel,
```

Comment:
you shouldn't need to pass both the robot_description and kmodel... they do the same thing